src/app.js

```diff
@@ -242,4 +242,4 @@ function locationError(err) {
 // Initial load
 console.log('Initial update');
 maincard.show();
-update();
```

Comment:
So that line is still there, as far as I can see. If you view the actual file, it's there. I believe this is because the new line has been removed.